addon/components/bs-dropdown/menu.js

```diff
@@ -85,38 +87,32 @@ export default class DropdownMenu extends Component {
    * @readonly
    * @private
    */
-  @computed
   get destinationElement() {
     return getDestinationElement(this);
   }
 
-  @computed('align')
   get alignClass() {
     return this.align !== 'left' ? `dropdown-menu-${this.align}` : undefined;
   }
 
-  @computed
-  get isOpen() {
-    return false;
-  }
+  @tracked
+  isOpen = this.args.isOpen;
 
-  set isOpen(value) {
-    // delay removing the menu from DOM to allow (delegated Ember) event to fire for the menu's children
-    // Fixes https://github.com/kaliber5/ember-bootstrap/issues/660
+  @action
+  updateIsOpen(value) {
     next(() => {
       if (this.isDestroying || this.isDestroyed) {
         return;
       }
-      this.set('_isOpen', value);
+      this.isOpen = value;
     });
-    return value;
   }
 
-  _isOpen = false;
   flip = true;
+
+  @tracked
   _popperApi = null;
```

Comment:
I struggle understanding what `_popperApi` is used for. I haven't found a place at which it's read. Neither a place at which it's set. Could you please give me a hint?